[WIP]: TR windturbine_anomaly_detection: Telegraf weld data ingestion

pooja-intel opened the pull request:
### Description

Changes:
 1.docker-compose.yml - Added changes to use weld data csv for mqtt
   publisher
 2. Telegraf_devmode.conf - Changed the topic name 
 3. time_series_analytics_microservice - Changes to use data from topic weld_data in tick script and udf.

Fixes # (issue)

### Any Newly Introduced Dependencies

Please describe any newly introduced 3rd party dependencies in this change. List their name, license information and how they are used in the project.

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

### Checklist:

- [ ] I agree to use the APACHE-2.0 license for my code changes.
- [ ] I have not introduced any 3rd party components incompatible with APACHE-2.0. 
- [ ] I have not included any company confidential information, trade secret, password or security token. 
- [ ] I have performed a self-review of my code.

pooja-intel committed:
windturbine_anomaly_detection: Telegraf weld data ingestion

Changes:
 1.docker-compose.yml - Added changes to use weld data csv for mqtt
   publisher
 2. Telegraf_devmode.conf - Changed the topic name
 3.time_series_analytics_microservice - Changes to use data from topic
   weld_data in tick script and udf.

Signed-off-by: Pooja Kumbharkar <pooja.kumbharkar@intel.com>

vkb1 commented on manufacturing-ai-suite/wind-turbine-anomaly-detection/time_series_analytics_microservice/tick_scripts/windturbine_anomaly_detector.tick:
```diff
@@ -1,22 +1,15 @@
                 .buffer(0)
                 .database('datain')
-                .measurement('wind_turbine_anomaly_data')
+                .measurement('weld_process_data')
```
Let's rename this to `weld_analytics_data`

vkb1 commented on manufacturing-ai-suite/wind-turbine-anomaly-detection/time_series_analytics_microservice/udfs/windturbine_anomaly_detector.py:
```diff
@@ -148,72 +148,84 @@ def process_the_point(x,y):
+
+        logger.info(f"Pressure: {pressure}, CO2 Weld Flow: {co2_weld_flow}, Feed: {feed}, \
+                    Primary Weld Current: {primary_weld_current}, Wire Consumed: {wire_consumed}, \
+                        Secondary Weld Voltage: {secondary_weld_voltage}")
```
Hey @pooja-intel ,
Thanks for quickly checking the ingestion part.
As you are working on understanding the spine app changes, you can introduce these changes as configs for the docker compose deployment of that spine app (will see on the feasibility later) if it's easily doable.

Post that you can prioritize the udf integration and the helm deployment stuff you can take it later.

pooja-intel committed:
windturbine_anomaly_detection: Folder restructure for apps

Changes:
 1.Makefile - Added app input parameter
 2.apps- Moved sample apps to apps

Signed-off-by: Pooja Kumbharkar <pooja.kumbharkar@intel.com>

pooja-intel committed:
windturbine_anomaly_detection: Reverted the udf changes

Changes:
 windturbine_anomaly_detector.py - Reverted the udf changes.

Signed-off-by: Pooja Kumbharkar <pooja.kumbharkar@intel.com>

pooja-intel committed:
windturbine_anomaly_detection: Reverted the windturbine udf changes

Signed-off-by: Pooja Kumbharkar <pooja.kumbharkar@intel.com>
